backend/de.metas.acct.base/src/main/sql/postgresql/ddl/views/c_orderline_id_with_missing_shipmentschedule_v.sql

```diff
@@ -0,0 +1,23 @@
+CREATE OR REPLACE VIEW c_orderline_id_with_missing_shipmentschedule_v AS
```

Comment:
please update the one we have (`backend/de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/ddl/views/C_OrderLine_ID_With_Missing_ShipmentSchedule_v.sql`)

Reply:
Missed that. Fixed.